prism/src/main/java/org/apache/falcon/resource/extensions/ExtensionManager.java

```diff
@@ -524,6 +541,26 @@ private void setEntityTags(Entity entity, String tags) {
         }
     }
 
+    private  JSONObject buildDetailResult(final String extensionName) throws FalconException {
+        ExtensionMetaStore metaStore = ExtensionStore.get().getMetaStore();
+
+        if (!metaStore.checkIfExtensionExists(extensionName)){
+            throw new StoreAccessException(new Exception("No extension resources found for " + extensionName));
+        }
+
+        ExtensionMetadataBean bean = metaStore.getDetail(extensionName);
+        JSONObject resultObject = new JSONObject();
+        try {
+            resultObject.put(EXTENSION_NAME, bean.getExtensionName().toLowerCase());
```

Comment:
why toLowerCase? Can you make changes to Bean to handle for the same so that all consumers are not worried about the same?